sys-devel/clang-common/clang-common-16.0.6-r2.ebuild

```diff
@@ -14,6 +14,7 @@ KEYWORDS="amd64 arm arm64 ~loong ppc ppc64 ~riscv sparc x86 ~amd64-linux ~ppc-ma
 IUSE="
 	default-compiler-rt default-libcxx default-lld llvm-libunwind
 	hardened
+	bootstrap-prefix
```

Comment:
We're wrapping at ~72 or ~80, or something like that, not putting one per line xP.

Reply:
I know. The 'default' line above felt so related to each other, that I decided to not break the pattern :)